src/ferc_xbrl_extractor/instance.py

```diff
@@ -172,6 +174,9 @@ def as_primary_key(self, filing_name: str) -> dict[str, str]:
                 "end_date": self.period.end_date,
             }
 
+        if optional_axis:
```

Comment:
I'm wondering if we should just leave these as null values as we put them into the dataframe. That would mean that, downstream, we'd have to interpret them somehow, but I think in our "convert XBRL to SQLite with minimal interpretation and processing" step we can avoid that interpretation step.

And then we don't have to have something in this very general "FactTable to DataFrame" function that understands the interpretation of every single nullable field in every table context.

Reply:
If we do want to interpret these values here, we should dynamically look up what the default value is, as defined in `FERC1_OPTIONAL_COLUMNS`.